plugins/tokens/update.js

```diff
@@ -61,8 +61,6 @@ module.exports = () => ({
                             .then(() => {
                                 const output = token.toJson();
 
-                                delete output.hash;
```

Comment:
no longer needed as this is handled by [Token.toJson](https://github.com/screwdriver-cd/models/blob/master/lib/token.js#L48)